package.json

```diff
@@ -4,6 +4,7 @@
   "homepage": "https://github.com/marionettejs/backbone.radio",
   "version": "2.0.0",
   "main": "build/backbone.radio.js",
+  "module": "build/backbone.radio.esm.js",
```

Comment:
The es  build must be in the package distributed.

Commit in the repository is optional. Anyway if deciding to commit to repository, It should be done only just before a release.